mobile/lib/utils/openapi_patching.dart

```diff
@@ -7,6 +7,11 @@ dynamic upgradeDto(dynamic value, String targetType) {
         if (value['rating'] == null) {
           value['rating'] = RatingResponse().toJson();
         }
+
+        if (value['download']['includeEmbeddedVideos'] == null) {
```

Comment:
why is this needed? It has a default value of false already.

```dart
class DownloadResponse {
  /// Returns a new [DownloadResponse] instance.
  DownloadResponse({
    required this.archiveSize,
    this.includeEmbeddedVideos = false,
  });
```

Reply:
So when you access the older server, the response from the old server doesn't have `includeEmbeddedVideos`